### DISCUSSION TITLE
Welcome to AeroCom experiments Discussions!

### DISCUSSION THREAD

**Discussion by duncanwp** [aerocom-community/experiments#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions about experiments.
  * Share experiment ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do (and hope to do) with this community.

**Comment by sarambl** [comment]
So, we'll primarily use the discussion for comments on things, right? Because commenting I think is more just a part of the committing and reviewing process of files.

**Reply by duncanwp** [reply]
Yeah, I think that makes sense. This could/should be more free-form, and we can still link to groups of issues / experiments

**Comment by sarambl** [comment]
What do you think about setting up one discussion per experiment? I put up one for the control experiment now, but I didn't want to just go ahead and make a mess if it's not wanted.

Another thing is that I am not sure we really advertised the github yet? Maybe something for an email or a brief advertisement during a forum?

**Reply by duncanwp** [reply]
Yeah we should definitely start advertising it now that it's taking shape. I don't really have a feel for the Discussion structure as I've not used it a lot. My feeling is that you need a bit of a critical mass for it to be useful so maybe we just keep it in one for now - especially since the conversations can be threaded. If it becomes too busy then maybe we split it out?
